locales/en-US/messages.properties

```diff
@@ -87,7 +87,7 @@ fileSavingFailedIndicator=Saving failed, retrying...
 help=Help
 deleteProjectLink=Delete Project
 exportProjectLink=Export (.zip)
-loginPromptEditor=<a href="#" title="Sign in to Thimble" id="login-link" class="navbar-login">Sign in</a> or <a href="#" title="Create an account" id="signup-link" class="navbar-login">Create an account</a> to Save and
+loginPromptEditor=<a title="Sign in to Thimble" id="login-link" class="navbar-login">Sign in</a> or <a title="Create an account" id="signup-link" class="navbar-login">Create an account</a> to Save and <publishBtn>Publish</publishBtn>
```

Comment:
Because apparently the publish button context is different when it is part of the "Sign in or Create an account to Save and" string, I had to add the publish button markup to this string. But because it was a lot of markup, and it will be hard for localizers to figure it out, I use `<publishBtn>` tags instead which I replace with the actual markup with nunjucks